MAUI/DataForm/Editing.md

```diff
@@ -0,0 +1,191 @@
+---
+layout: post
+title: Data handling in .NET MAUI DataForm control | Syncfusion
+description: Learn here all about the commit feature in Syncfusion .NET MAUI DataForm (SfDataForm) control and more.
+platform: maui
+control: SfDataForm
+documentation: ug
+---
+
+# Commit in .NET MAUI DataForm (SfDataForm)
+
+The data form commits the data and user input in order to update the value in the underlying data object.
+
+## Commit mode
+
+The [CommitMode] determines when the value should be committed to the underlying data object to allow users commit only correct values.
+
+The supported commit modes are as follows:
+
+* LostFocus
+* PropertyChanged
+* Manual
+
+#### LostFocus
+
+If the commit mode is `LostFocus`, the value is committed when the editor lost its focus. By default DataForm [CommitMode] is `LostFocus`.
+
+{% tabs %}
+{% highlight XAML %}
+<ContentPage 
+...
+             xmlns:dataForm="clr-namespace:Syncfusion.Maui.DataForm;assembly=Syncfusion.Maui.DataForm">
+    <dataForm:SfDataForm
+        x:Name="dataForm"
+        CommitMode="LostFocus">
+    </dataForm:SfDataForm>
+</ContentPage>
+{% endhighlight %}
+{% highlight C# %}
+this.dataForm.CommitMode = DataFormCommitMode.LostFocus;
+{% endhighlight %}
+{% endtabs %}
+
+#### PropertyChanged
+
+If the commit mode is `PropertyChanged`, the value will be committed immediately when it is changed.
+
+{% tabs %}
+{% highlight XAML %}
+<ContentPage 
+...
+             xmlns:dataForm="clr-namespace:Syncfusion.Maui.DataForm;assembly=Syncfusion.Maui.DataForm">
+    <dataForm:SfDataForm
+        x:Name="dataForm"
+        CommitMode="PropertyChanged">
+    </dataForm:SfDataForm>
+</ContentPage>
+{% endhighlight %}
+{% highlight C# %}
+this.dataForm.CommitMode = DataFormCommitMode.PropertyChanged;
+{% endhighlight %}
+{% endtabs %}
+
+#### Manual
+
+If the commit mode is `Manual`, the value should be committed manually by calling the [SfDataForm.Commit] method.
```

Comment:
If the commit mode is `Manual,` the value should be committed manually by calling the [SfDataForm.Commit] method.

Reply:
content changed